src/db.js

```diff
@@ -15,18 +15,13 @@ const {
   POSTGRES_DB,
 } = process.env;
 
-/*
-  All credentials setted to default values (exsept password - it is exapmle)
-  replace if needed with your own
-*/
-
 const sequelize = new Sequelize({
```

Comment:
You are creating a new Sequelize instance directly in this file. This is not a good practice as it makes the code less modular and more difficult to test. It would be better to do this in a separate factory function, which you then import in this file.